roles/ec2-s3-website/tasks/create.yml

```diff
@@ -71,4 +71,25 @@
         command: create
         overwrite: true
       with_items: "{{ s3_site_data.aliases|default([]) }}"
-  when: not skip_route53|default(False)
\ No newline at end of file
+  when: not skip_route53|default(False)
+
+- name: "Output our s3 default url"
+  debug:
+    msg: "http://{{ s3_site_data.bucket.name }}.s3-website.{{ s3_site_data.region }}.amazonaws.com"
+
+- name: "Output our s3 alias urls"
+  debug:
+    msg: "http://{{ item }}.s3-website.{{ s3_site_data.region }}.amazonaws.com"
```

Comment:
alias is like this too.?

Reply:
yeah, example

```
name: maintenance.dev.my.domain (ie top level, covers all *.dev)
alias: 
 - maintenance.my-target.dev.my.domain
```
Would create 2 buckets, the alias pointing to the name, also has separate entries in s3 then